docs/authentication/social-connections/tiktok.mdx

```diff
@@ -1,47 +1,75 @@
 ---
 title: Add TikTok as a social connection
-description: Learn how to set up social connection with TikTok.
+description: Learn how to allow users to sign up and sign in to your Clerk app with their TikTok account using OAuth.
 ---
 
-How to set up social connection with TikTok
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "A Clerk app is required.",
+      link: "/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "A TikTok Developer account is required.",
+      link: "https://developers.tiktok.com",
+      icon: "user-circle",
+    }
+  ]}
+>
+  - Use TikTok ID to authenticate users with OAuth.
+</TutorialHero>
 
-## Overview
+Enabling OAuth with [TikTok](https://developers.tiktok.com/doc/login-kit-manage-user-access-tokens) allows your users to sign up and sign in to your Clerk app with their TikTok account.
 
-Adding social connection with TikTok to your app with Clerk is done in a few steps - you only need to set the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.
+> [!IMPORTANT]
+> Due to TikTok's requirement to verify URL ownership for all redirect URLs, TikTok cannot be used as a social connection in development environments. This is because individual users cannot verify ownership of the Clerk development URL (`accounts.dev`). However, TikTok can be used properly in production environments where you can verify your own URLs. For more information, see [TikTok's URL ownership guidelines](https://developers.tiktok.com/doc/getting-started-create-an-app#verify_url_ownership).
```

Comment:
What do we think about doing this:
```suggestion
## Configure for your development instance

Due to TikTok's requirement to verify URL ownership for all redirect URLs, **TikTok cannot be used as a social connection in development environments.** This is because individual users cannot verify ownership of the Clerk development URL (`accounts.dev`). For more information, see [TikTok's URL ownership guidelines](https://developers.tiktok.com/doc/getting-started-create-an-app#verify_url_ownership).

## Configure for your production instance
```

Reply:
Also, if this is true, then why can we set it in the Dashboard without providing custom credentials?
Who gave you this info? We need to ask them what's going on with that


https://github.com/user-attachments/assets/243d980a-8a6a-4687-bcb3-cfacbc371c2f